velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -411,6 +411,14 @@ TEST_F(DecimalArithmeticTest, round) {
       {makeFlatVector<int128_t>(
           {DecimalUtil::kLongDecimalMax, DecimalUtil::kLongDecimalMin},
           DECIMAL(38, 1))});
+  testDecimalExpr<TypeKind::BIGINT>(
+      {makeFlatVector<int64_t>(
+          {3000000000000000, -3000000000000000}, DECIMAL(17, 0))},
+      "round(c0)",
+      {makeFlatVector<int128_t>(
+          {facebook::velox::HugeInt::parse("3000000000000000000000"),
+           facebook::velox::HugeInt::parse("-3000000000000000000000")},
```

Comment:
Thanks for the fix! The prefixes `facebook::velox::` could be dropped, and maybe use `"3" + std::string(n, '0')` to create a string for readability.

Reply:
Thank you, it has been modified.